test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -285,3 +285,28 @@ baz
         }
     }
 }
+
+Describe "Get-Content -Raw test" -Tags "CI" {
```

Comment:
This test code is a good candidate to be rewritten using `-testcases` construct, as per [testing guidelines](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#parameter-generation).